tests/integrations/test_dbm.py

```diff
@@ -31,12 +31,56 @@ def setup_trace_payload(self):
                 weblog.get("/dbm", params={"integration": "mysqli"}),
             ]
 
-    def test_trace_payload(self):
+    def find_db_spans(self):
+        self.db_span = None
         for r in self.requests:
-            assert r.status_code == 200, f"{r.request.url} is not successful"
-            for _, _, span in interfaces.library.get_spans(request=r):
-                if span.get("span_type") != "sql":
-                    return
+            assert r.status_code == 200, f"Request: {r.request.url} wasn't successful."
+            for _, trace in interfaces.library.get_traces(request=r):
```

Comment:
`get_spans` can replace the nested for

```suggestion
            for _, _, span in interfaces.library.get_spans(request=r):
```